cfme/infrastructure/config_management/__init__.py

```diff
@@ -414,32 +389,35 @@ def rest_api_entity(self):
             name=self.ui_name
         ).provider_id
 
-        return RestEntity(
-            self.appliance.rest_api.collections.providers,
-            data={
-                "href": self.appliance.url_path(
-                    "/api/providers/{}?provider_class=provider".format(provider_id)
-                )
-            },
-        )
+        return RestEntity(self.appliance.rest_api.collections.providers, data={
+            "href": self.appliance.url_path(
+                "/api/providers/{}?provider_class=provider".format(provider_id)
+            )
+        })
 
+    # TODO: implement this via Sentaku
     def create_rest(self):
         """Create the config manager in CFME using REST"""
+        include_ssl = False
         if "ansible_tower" in self.key:
             config_type = "AnsibleTower"
         elif "satellite" in self.key:
             config_type = "Foreman"
+            include_ssl = True
 
         payload = {
             "type": "ManageIQ::Providers::{}::Provider".format(config_type),
             "url": self.url,
             "name": self.name,
-            "verify_ssl": self.ssl,
             "credentials": {
                 "userid": self.credentials.view_value_mapping["username"],
                 "password": self.credentials.view_value_mapping["password"],
             },
         }
+
+        if include_ssl:
```

Comment:
As we need `verify_ssl` only for `satellite`. better to put check here only l.e. `if "satellite" in self.key`; no need of above selections of `include_ssl`

Reply:
This in part has to do with the `payload` rather than repeating that `if` statement twice, I thought it would be better to just define that variable 

```
        if "ansible_tower" in self.key:
            config_type = "AnsibleTower"
        elif "satellite" in self.key:
            config_type = "Foreman"
            include_ssl = True
```